src/routing/Router.svelte

```diff
@@ -47,25 +47,30 @@
 
   function useComponent(componentToUse, view) {
     return function handle({ params: { product, section, probeName } }) {
+      const storeValue = get(store);
```

Comment:
Does `$store` not work in this case?

Reply:
in hindsight I was trying to figure out if this issue was caused by autosubscriptions colliding with `page.js`. I think the source of the issue turned out to be elsewhere, so we could revert this.